internal/go-prompt/prompt.go

```diff
@@ -130,21 +130,21 @@ func (p *Prompt) feed(b []byte) (shouldExit bool, exec *Exec) {
 		exec = &Exec{input: p.buf.Text()}
 		p.buf = NewBuffer()
 		if exec.input != "" {
-			p.history.Add(exec.input)
+			p.History.Add(exec.input)
 		}
 	case ControlC:
 		p.renderer.BreakLine(p.buf)
 		p.buf = NewBuffer()
-		p.history.Clear()
+		p.History.Clear()
 	case Up, ControlP:
 		if !completing { // Don't use p.completion.Completing() because it takes double operation when switch to selected=-1.
```

Comment:
Could you move these kinds of comments to the line below?

Reply:
This and similar comments moved to a new line https://github.com/hazelcast/hazelcast-commandline-client/pull/66/commits/fe6bf1e1b2ca7dd456aff72357e77644aca95af7